ToDo/ToDo/Services/ToDo/IToDoRespository.cs

```diff
@@ -0,0 +1,15 @@
+﻿using ToDo.Model;
+
+namespace ToDo.Services.ToDo
+{
+    public interface IToDoRespository
```

Comment:
```suggestion
    public interface IToDoService
```
Service sẽ thực hiện việc xử lý nghiệp vụ và query dữ liệu trong DB nhé